### ISSUE TITLE
Weird issue with Microsoft Teams

### ISSUE THREAD

**Issue by ShaharHD** [kofigumbs/multi#75]
Trying to use with https://teams.microsoft.com after login I'm stuck with this screen while Cmd-R to refresh or closing and re-opening doesn't resolve the issue.

Any idea on how to resolve this?

<img width="1047" alt="image" src="https://user-images.githubusercontent.com/700985/158679863-c5352968-5e6c-4fa1-b84e-8c123208a6ce.png">

**Comment by ShaharHD** [comment]
So inspect element did give some more interesting insight.

The reason I'm getting the screen I'm getting is there seems to be some elements which Safari does not implement, specific to how Microsoft is handling MFA (when I login to teams with that account I have my MS authenticator popping up on my phone with request to authenticate).

Could it be related to this https://support.microsoft.com/en-us/office/safari-browser-support-1aac0a7c-35a8-42c1-a7df-f674afe234df#:~:text=If%20you%20want%20to%20use,.microsoft.com%20in%20Safari. ?

Is it possible to perform this via the config file?

**Comment by ShaharHD** [comment]
I'm closing the issue as now Teams is native apple silicon.

FYI the block list workaround didn't help.

**Comment by ratonhat** [comment]
As of know you can use Teams in Safari, but you need to disable `Cross site tracking`. Is there any way we could also disable this in Multi?

**Comment by ShaharHD** [comment]
> As of know you can use Teams in Safari, but you need to disable `Prevent-Cross site tracking`. Is there any way we could also disable this in Multi?

maybe create another issue?
would love to see if it works - as then I can run in parallel multiple instances of Teams (until they finally allow to do like Slack where you can be logged in to multiple accounts in parallel)
